test/test_saemref.py

```diff
@@ -34,3 +63,22 @@ def test_idempotence(Salt, state, exclude):
     for _, item in result.items():
         assert item["result"] is True
         assert item["changes"] == {}
+
+
+@wait_saemref_started
+def test_saemref_running(Process, Service, Socket, Command):
+    assert Service("supervisord").is_enabled
+
+    supervisord = Process.get(comm="supervisord")
+    assert supervisord.user == "saemref"
+    assert supervisord.group == "saemref"
+
+    cubicweb = Process.get(ppid=supervisord.pid)
+    assert cubicweb.comm == "cubicweb-ctl"
+    assert cubicweb.user == "saemref"
+    assert cubicweb.group == "saemref"
+
+    assert Socket("tcp://0.0.0.0:8080").is_listening
+
+    html = Command.check_output("curl http://localhost:8080")
+    assert "<title>accueil (Référentiel SAEM)</title>" in html
```

Comment:
Added an explanation in the commits messsage (2ddf7ab75a4d07a8036c4a5e4951ad8481fa2f0c)